integration-tests/api/__tests__/admin/order.js

```diff
@@ -31,8 +34,13 @@ describe("/admin/orders", () => {
 
   beforeAll(async () => {
     const cwd = path.resolve(path.join(__dirname, "..", ".."))
-    dbConnection = await initDb({ cwd })
-    medusaProcess = await setupServer({ cwd })
+    const [serverProcess, connection] = await startServerWithEnvironment({
+      cwd,
+      env: { MEDUSA_FF_SALES_CHANNELS: true },
+      verbose: false,
+    })
```

Comment:
todo: the feature flag should only be enabled for the tests related to sales channels. So all the existing tests in this file should run under the "old" version, while only the tests that use the sales channels should have the feature flag applied.

Reply:
Would it be alright if I create a sub describe where I would do that?

Reply:
Yeah - have no strong opinions on how this should be structured as long as we have all existing test running without the flag :)